cms/celery.py

```diff
@@ -9,7 +9,6 @@
 import os
 
 from celery import Celery
-from django.conf import settings
```

Comment:
I can't comment on the actual line, but probably change the default value for `DJANGO_SETTINGS_MODULE` in here too?